.github/workflows/dispatch.yml

```diff
@@ -7,45 +7,59 @@ on:
     branches: [ master, testing ]
 
 env:
-  # We could use a higher number here. We use a small number just to make sure to create multiple batches.
-  MAX_REPOS_PER_WORKFLOW: 16
-  FILES: '[ ".github/workflows/automerge.yml", ".github/workflows/go-test.yml", ".github/workflows/go-check.yml" ]' # a JSON array of the files to distribute
+  # Number of repositories in a batch.
+  # Deployment jobs in the same batch use the same concurrency group,
+  # and are therefore run sequentially.
+  # With ~180 repos, this means we'll run around 9 instances of the copy workflow in parallel.
+  # This is (hopefully) sufficient to prevent us from triggering GitHub Action's abuse detection mechanism.
+  MAX_REPOS_PER_WORKFLOW: 20
 
 jobs:
   matrix:
     name: Trigger copy workflows
     runs-on: ubuntu-latest
     outputs:
-      targets: ${{ steps.set-matrix.outputs.targets }}
+      batches: ${{ steps.set-matrix.outputs.batches }}
     steps:
       - uses: actions/checkout@v2
       - id: set-matrix
         run: |
-          CONFIG=config.json
-          if [[ $GITHUB_REF == refs/heads/testing ]]; then
-            CONFIG=config-testing.json
-          fi
-          TARGETS=$(jq '. | _nwise(${{ env.MAX_REPOS_PER_WORKFLOW }})' $CONFIG | jq -sc '. | to_entries')
-          echo "::set-output name=targets::$TARGETS"
+          TARGETS=()
+          for config in configs/*.json; do
+            if [[ $GITHUB_REF == refs/heads/testing && $config != configs/testing.json ]]; then
+              continue
+            fi
+            if [[ $GITHUB_REF != refs/heads/testing && $config == configs/testing.json ]]; then
+              continue
+            fi
+            DEFAULTS=$(jq -c '.defaults' $config)
+            TARGETS+=($(jq -c ".repositories[] | $DEFAULTS + ." $config))
```

Comment:
If a specific `key` is present in the target object, it will override the one from defaults.

Reply:
Add something along these lines as a comment?